rcl/test/rcl/test_subscription_content_filter_options.cpp

```diff
@@ -0,0 +1,268 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
```

Comment:
nit

```suggestion
// Copyright 2022 Open Source Robotics Foundation, Inc.
```

Reply:
updated in 0fcdc5b9a58a4e2f15dc3784abb8ff81e2f2f099